.evergreen-release.yml

```diff
@@ -147,7 +147,7 @@ buildvariants:
       - name: "*"
         variant: preflight_release_images
     run_on:
-      - ubuntu2404-small
+      - release-ubuntu2404-small # This is required for CISA attestation https://jira.mongodb.org/browse/DEVPROD-17780
```

Comment:
Hi @MaciejKaras,
Changing this configuration here makes our `.evergreen.yml` configuration invalid (and our CI doesn't run) with below error

```
sudo evergreen validate --path .evergreen.yml -p mongodb-kubernetes
....
ERROR: task 'run_conditionally_prepare_and_upload_openshift_bundles' in buildvariant 'prepare_openshift_bundles' runs on a single task distro 'release-ubuntu2404-small' and cannot use the generate tasks command
.evergreen.yml is an invalid configuration
```

Is it ok if I go ahead and merge this PR without this `run_on` change.